salt/utils/event.py

```diff
@@ -223,17 +226,54 @@ def __load_uri(self, sock_dir, node):
         )
         return puburi, pulluri
 
-    def subscribe(self, tag=None):
+    def subscribe(self, tag):
```

Comment:
Tag could be assumed empty string (and startswith match type in 2015.8) - but this would be odd usage because it would then log every single pending event it receives, until unsubscribed.

Looking at history of this function though, and the fact before this PR it was a No-Op, I wonder if I really should have set up new functions for setting up event caching, rather than using these ones, and left these ones as No-Ops.